<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Grupo</class>
 <widget class="QWidget" name="Grupo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="msgView">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>155</y>
     <width>580</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>580</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>300</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="msgWrite">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>461</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="msgSend">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>27</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>400</height>
    </size>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_profilePicture">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>180</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Practica 7/Icons/Icon_Fish.jpg</normaloff>../Practica 7/Icons/Icon_Fish.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>165</width>
     <height>165</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="peopleList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>230</width>
     <height>220</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="groupName">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>110</y>
     <width>580</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>370</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name_Group</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="usersName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>230</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>580</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Activos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Close">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>530</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
